scss/partials/_typography.scss

```diff
@@ -0,0 +1,155 @@
+.p {
+  @extend p;
+
+  &--lead,
+  &--lead-1 {
```

Comment:
I think this style syntax is where we are running into verbosity issues. Having this style nesting with the `@extend`above  means every sub class is fully rearticulating the base `p` styling. I think the fix here is to do the extension on a single selector for `[class^=p]` like I described with headers, and use the nesting with `&` notation just for the purposes of code structure and class concatenation.